perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_entityless_attribute_def_def_identityAlertsTemplates.java

```diff
@@ -0,0 +1,133 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.AttributesManager;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.EntitylessAttributesModuleAbstract;
+
+import java.time.format.DateTimeFormatter;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+/**
+ * Templates, for identity changes alerts.
+ *
+ * @author vojtech.sassmann@gmail.com
+ */
+public class urn_perun_entityless_attribute_def_def_identityAlertsTemplates extends EntitylessAttributesModuleAbstract {
+
+	public static final String UES_ADDED_PREFERRED_MAIL = "uesAddedPreferredMail";
+	public static final String UES_ADDED_PREFERRED_MAIL_SUBJECT = "uesAddedPreferredMailSubject";
+	public static final String UES_ADDED_UES_MAIL = "uesAddedUESMail";
+	public static final String UES_ADDED_UES_MAIL_SUBJECT = "uesAddedUESMailSubject";
+
+	public static final String UES_REMOVED_PREF_MAIL = "uesRemovedPreferredMail";
+	public static final String UES_REMOVED_PREF_MAIL_SUBJECT = "uesRemovedPreferredMailSubject";
+	public static final String UES_REMOVED_UES_MAIL = "uesRemovedUESMail";
+	public static final String UES_REMOVED_UES_MAIL_SUBJECT = "uesRemovedUESMailSubject";
+
+	public static final String LOGIN_PLACEHOLDER = "\\{login}";
+	public static final String ORG_PLACEHOLDER = "\\{organization}";
+	public static final String TIME_PLACEHOLDER = "\\{time}";
+
+	public static final String ORG_UNKNOWN_TEXT = "<unknown>";
+	public static final DateTimeFormatter ALERT_TIME_FORMATTER = DateTimeFormatter.ofPattern("MM/dd/yyyy - HH:mm:ss z");
+
+	public static final String KEY_EN = "en";
+
+	public static final String DEFAULT_IDENTITY_ADDED_PREF_MAIL_SUBJECT = "New identity added";
+	public static final String DEFAULT_IDENTITY_ADDED_UES_MAIL_SUBJECT = "Account linked";
+	public static final String DEFAULT_IDENTITY_ADDED_PREF_MAIL = """
+			A new identity has been added to your account. If you don't recognize this activity, please contact our support at perun@cesnet.cz.
```

Comment:
Just a note: I'd like to have this mail address configurable too, but we probably don't have any shared mail config (from/support mail). It`s partly divided between core and gui config. We should unify those settings where possible and then re-use it. I'll plan this for the upcoming cycles.

Reply:
Ok, for now, the email could be just changed with the whole template in the entityless attribute.